examples/human/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM ubuntu:24.04
+
```

Comment:
We typically handle Python dependencies by just inheriting from `python:3.12-bookworm`, would that work here?